src/Nirum/Targets/Python.hs

```diff
@@ -1104,36 +1121,33 @@ class $className({T.intercalate "," $ compileExtendClasses annotations}):
             else:
                 name = attribute_name
             tag_types = cls.__nirum_tag_types__
-            if callable(tag_types):  # old compiler could generate non-callable map
+            # old compiler could generate non-callable map
+            if callable(tag_types):
                 tag_types = dict(tag_types())
             try:
                 args[name] = deserialize_meta(tag_types[name], item)
```

Comment:
Despite this kind of conditionals are necessary for the runtime library to be compatible old and new versions of the compiler, it's no more needed for compiler-generated codes; it's guaranteed that it's always generated by the new one.